types/react-native-fbsdk/index.d.ts

```diff
@@ -0,0 +1,1089 @@
+// Type definitions for react-native-fbsdk 0.6
+// Project: https://github.com/facebook/react-native-fbsdk
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { ComponentClass, Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+// tslint:disable-next-line:export-just-namespace
+export = FBSDK;
+
+declare namespace FBSDK {
+    /**
+     * Commonly used Facebook permissions
+     */
+    type Permissions = 'public_profile'
+    | 'user_friends'
+    | 'email'
+    | 'user_about_me'
+    | 'user_actions.books'
+    | 'user_actions.fitness'
+    | 'user_actions.music'
+    | 'user_actions.news'
+    | 'user_actions.video'
+    | 'user_birthday'
+    | 'user_education_history'
+    | 'user_events'
+    | 'user_games_activity'
+    | 'user_hometown'
+    | 'user_likes'
+    | 'user_location'
+    | 'user_managed_groups'
+    | 'user_photos'
+    | 'user_posts'
+    | 'user_relationships'
+    | 'user_relationship_details'
+    | 'user_religion_politics'
+    | 'user_tagged_places'
+    | 'user_videos'
+    | 'user_website'
+    | 'user_work_history'
+    | 'read_custom_friendlists'
+    | 'read_insights'
+    | 'read_audience_network_insights'
+    | 'read_page_mailboxes'
+    | 'manage_pages'
+    | 'publish_pages'
+    | 'publish_actions'
+    | 'rsvp_event'
+    | 'pages_show_list'
+    | 'pages_manage_cta'
+    | 'pages_manage_instant_articles'
+    | 'ads_read'
+    | 'ads_management'
+    | 'business_management'
+    | 'pages_messaging'
+    | 'pages_messaging_subscriptions'
+    | 'pages_messaging_payments'
+    | 'pages_messaging_phone_number'
+    | string;
+
+    /**
+     * Specifies the privacy of a group.
+     */
+    type AppGroupPrivacy =
+        // Anyone can see the group, who's in in and what members post.
+        'Open' |
+        // Anyone can see the group and who's in it, but only members can see posts.
+        'Closed';
+
+    /**
+     * A model for app invites.
```

Comment:
Yeah, this comment seems copy-pasted from the interface below. Could it be corrected?